python/ml4ir/applications/ranking/tests/test_losses.py

```diff
@@ -28,38 +28,49 @@ def setUp(self):
 
     def test_sigmoid_cross_entropy(self):
         """Test the sigmoid cross entropy pointiwse loss object"""
-        loss = pointwise_losses.SigmoidCrossEntropy()
-        activation_op = loss.get_final_activation_op(output_name="y_pred")
-        loss_fn = loss.get_loss_fn(mask=self.mask)
-
-        y_pred = activation_op(logits=self.logits, mask=self.mask)
+        loss_op = pointwise_losses.SigmoidCrossEntropy()
+
+        y_pred = loss_op.final_activation_op({
+            "logits": self.logits,
+            "metadata": {
+                "mask": self.mask
+            }
+        })
         assert np.isclose(y_pred[0][0].numpy(), 0.54905695, atol=1e-5)
-        assert np.isclose(y_pred[2][4].numpy(), 0.64832306, atol=1e-5)
+        assert np.isclose(y_pred[2][4].numpy(), 0., atol=1e-5)
```

Comment:
Why did this change?

Reply:
I think this should be expected to change really because we've modified underlying tensorflow, numpy versions which could be impacting the random seeds or even the implementations. This is my guess though.

Reply:
(Ignore my previous reply)

This is 0., and should be 0., because we mask the predictions too. We weren't doing this earlier. It wasn't a breaking issue though because it was being masked in the loss. I'm doing the masking in the activation now to make sure in the next version update, we can remove passing input features to the loss function like we discussed.

Reply:
If you notice, the final loss computed is still the same in the next line.